language/src/regent/std.t

```diff
@@ -3435,11 +3433,72 @@ function std.setup(main_task, extra_setup_thunk, registration_name)
   return main, names
 end
 
+-- Generate every n-th task wrapper in this process (all of them if n == 1),
+-- the compiler will pick them up automatically.
+local function make_task_wrappers(n, i)
+  n = n or 1
+  i = i or 1
+  local task_wrappers = {}
+  for j,variant in ipairs(variants) do
+    if (j-1) % n == (i-1) then
+      task_wrappers[variant:wrapper_name()] = variant:make_wrapper()
+    end
+  end
+  return task_wrappers
+end
+
+local function compile_tasks_in_parallel()
+  -- Force codegen; the main process will need to codegen later anyway, so we
+  -- might as well do it now and not duplicate the work on the children.
+  for _,variant in ipairs(variants) do
+    variant.task:complete()
+  end
+
+  -- Spawn children processes, assign a subset of tasks to each to compile.
+  -- HACK: Terra functions used by more than one task may get compiled
+  -- multiple times, and included in multiple object files by different
+  -- children. They should all be exactly the same, so the linker may choose
+  -- to keep any one of them.
+  local flags = terralib.newlist({'-Wl,--allow-multiple-definition'})
```

Comment:
Is this still needed with the Terra change?

Edit: Is fixed in a later commit. Would be nice to reorganize the patches to avoid this.